ns/math.n3

```diff
@@ -0,0 +1,239 @@
+# Schema for CWM's mathematical built-ins, SBP 2001-12
+#
+# $Id: math.n3,v 1.12 2010-03-30 15:18:08 timbl Exp $
+
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix log: <http://www.w3.org/2000/10/swap/log#> .
+@prefix math: <http://www.w3.org/2000/10/swap/math#> .
+@prefix string: <http://www.w3.org/2000/10/swap/string#> .
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+
+math:StrictProperty a rdfs:Class;
+    rdfs:label "StrictProperty";
+    rdfs:subClassOf rdf:Property.
+
+math:Value a rdfs:Class; rdfs:label "Value"; 
```

Comment:
This should probably be `rdfs:subClassOf xsd:anyAtomicType`, or be specific about `xsd:decimal`, `xsd:float`, `xsd:integer`, `xsd:double`, and `xsd:string`.

Reply:
Unsure whether we need it really ..